content/blog/wingingit/winging-it-17.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Revisiting Fluid Type
+episode: 17
+tags:
+  - CSS
+  - Fonts
+  - Layout
+  - Typography
+author:
+  - miriam
+  - stacy
+date: 2025-03-20
+length: 57 mins
+image:
+  src: winging-it/winging-it-17.jpg
+media:
+  - youtube: py41Ys-iRvk
+summary: |
+  Miriam's been asking questions
+  about fluid typography,
+  and how a website design should plan for
+  (and respond to) user preferences.
+  And we're not the only ones thinking about it!
+  This month we're going to
+  continue the conversation,
+  with the experts
+  from [Clearleft](https://clearleft.com/)
+  behind excellent resources
+  like [Utopia.fyi](https://utopia.fyi/)
+  and [WebTypography.net](https://webtypography.net/]).
+---
+
+{% callout 'note', false %}
+Check out our [Winging It](/wingingit/)
+conversations about design, frontend,
+and backend development.
+
+**Winging It** episode 15: [Relative Units & Typography](/2025/01/24/winging-it-15/)
+{% endcallout %}
+
+{% import "embed.macros.njk" as embed %}
+{% import "utility.macros.njk" as utility %}
+
+{{ embed.figure(
+  data=media
+) }}
+
+{{ utility.main_action(
+  'Subscribe to Channel »',
+  subscribe_url
+) }}
+
+ ## What we cover:
+
+- How do we balance author and user intents?
+- Adrian Roselli’s "ultimate ideal" base font-size
+- Possible solutions/ideas to achieve desired font-size (per site)
+- What does an `em` represent?
+- Common approach to fluid type with `vi` & `clamp()`
+- Utopia helps, and catches potential issues with `vi`
+- How can designers work with fluid typography in traditional design tools
```

Comment:
@SondraE I added/edited a few of these
@mirisuzanne does this look right to you as well?